commit 7182d3e7d17722d088322695fc09f0d3bb7f1eab
Author: John Daley <johndale@cisco.com>
Date:   Wed Mar 30 11:07:31 2016 -0700

    enic: expose Rx missed packets counter
    
    Update the 'imissed' counter with the number of packets dropped
    by the NIC.
    
    Fixes: fefed3d1e62c ("enic: new driver")
    
    Signed-off-by: John Daley <johndale@cisco.com>
    Reviewed-by: Nelson Escobar <neescoba@cisco.com>

diff --git a/drivers/net/enic/enic_main.c b/drivers/net/enic/enic_main.c
index 2f79cf0..e3da51d 100644
--- a/drivers/net/enic/enic_main.c
+++ b/drivers/net/enic/enic_main.c
@@ -246,6 +246,8 @@ void enic_dev_stats_get(struct enic *enic, struct rte_eth_stats *r_stats)
 	r_stats->ierrors = stats->rx.rx_errors;
 	r_stats->oerrors = stats->tx.tx_errors;
 
+	r_stats->imissed = stats->rx.rx_drop;
+
 	r_stats->imcasts = stats->rx.rx_multicast_frames_ok;
 	r_stats->rx_nombuf = stats->rx.rx_no_bufs;
 }
